pkg/connection/connection.go

```diff
@@ -71,9 +66,14 @@ func CheckAuthFromContext(ctx context.Context, runPath string) error {
 
 // Dial creates a gRPC connection to the server.
 func Dial(ctx context.Context, address, passwordFile string) (proto.ManageClient, func() error, error) {
-	pw, err := os.ReadFile(passwordFile)
-	if err != nil {
-		return nil, nil, fmt.Errorf("reading manage auth password file %q: %w", passwordFile, err)
+	pw := []byte(shared.DevelopmentPassword)
+	dev, _ := strconv.ParseBool(os.Getenv("OPENSLIDES_DEVELOPMENT"))
```

Comment:
Maybe add a comment, that you don't check the error, because for any invalid value dev will be "false", and this is the expected behavior anyway.

Reply:
Done.